test/test_models.py

```diff
@@ -36,18 +36,20 @@ def test_mfcc(self):
 class TestMelResNet(common_utils.TorchaudioTestCase):
 
     def test_waveform(self):
+        """Validate the output dimensions of a _MelResNet block.
+        """
 
-        batch_size = 2
-        num_features = 200
-        input_dims = 100
-        output_dims = 128
-        res_blocks = 10
-        hidden_dims = 128
-        pad = 2
+        n_batch = 2
+        n_time = 200
+        n_freq = 100
+        n_output = 256
+        n_res_block = 10
+        n_hidden = 128
+        kernel_size = 5
```

Comment:
nit: 5 changed because of definition change from pad to kernel_size?

Reply:
Yes, `kernel_size = pad * 2 + 1`